web-platform/website/src/components/navigation-bar/NavigationBar.tsx

```diff
@@ -126,4 +125,14 @@ const styles = {
       },
     }
   ),
+  logoCol: style(
+    {
+      marginRight: `0`,
```

Comment:
You can just tag me-0 on line 30. If we need this style we can just use numeric for marginRight as in marginRight: 0